src/audio.rs

```diff
@@ -92,7 +92,7 @@ impl Audio {
     }
 
     fn data_from_effect(&mut self, effect: Effect) -> SoundData {
-        use Effect::*;
+        use Effect::{Click, Explosion, GameOver, MonsterHit, MonsterMoved, Walk};
```

Comment:
The glob imports are an explicit choice here. I don't do top-level glob imports, but I absolutely do like them withing a given function or a scope. I would actually prefer using e.g. `Effect::Click` everywhere in the code to explicitly writing out all the enum variant like here.

Code like this should remain unchanged.

Reply:
I will revert it.